__tests__/db.test.js

```diff
@@ -52,8 +106,219 @@ describe("GET /api", () =>{
                 expect(body[endPoint]).toHaveProperty("description")
                 expect(body[endPoint]).toHaveProperty("queries")
                 expect(body[endPoint]).toHaveProperty("exampleResponse")
+            })
+            })
+        })
+    })
+describe("GET /api/articles", () =>{
+    test("Returns 200 status and an array", () =>{
+        return request(app)
+        .get('/api/articles')
+        .then((res)=>{
+            expect(res.statusCode).toBe(200)
+            expect(Array.isArray(res.body.articles)).toBe(true)
+        })
+    })
+    test("All objects in the articles array contain basic article data, without body", () =>{
+        return request(app)
+        .get('/api/articles')
+        .then(({body})=>{
+            expect(body.articles.length > 0).toBe(true)
+            body.articles.forEach((article)=>{
+                expect(article).toHaveProperty("article_id")
+                expect(article).toHaveProperty("author")
+                expect(article).toHaveProperty("title")
+                expect(article).toHaveProperty("topic")
+                expect(article).toHaveProperty("created_at")
+                expect(article).toHaveProperty("votes")
+                expect(article).toHaveProperty("article_img_url")
+                expect(article).not.toHaveProperty('body')
+            })
+        })
+    })
+    test("Articles contain comment_count, a numeric count of comments referencing that ID within the comments table", () =>{
+        return request(app)
+        .get('/api/articles')
+        .then(({body})=>{
+            expect(body.articles.length > 0).toBe(true)
+            body.articles.forEach((article)=>{
+                expect(article).toHaveProperty("comment_count", expect.any(Number))
+            })
+        })
+    })
+    test("Articles return sorted by their date in decending order", () =>{
+        return request(app)
+        .get('/api/articles')   
+        .then(({body})=>{
+            expect(new Date(body.articles[0].created_at).getTime() >= new Date(body.articles[1].created_at).getTime()).toBe(true)
+            expect(new Date(body.articles[1].created_at).getTime() >= new Date(body.articles[2].created_at).getTime()).toBe(true)
+            expect(new Date(body.articles[2].created_at).getTime() >= new Date(body.articles[3].created_at).getTime()).toBe(true)
+            expect(new Date(body.articles[3].created_at).getTime() >= new Date(body.articles[4].created_at).getTime()).toBe(true)
+            expect(new Date(body.articles[4].created_at).getTime() >= new Date(body.articles[6].created_at).getTime()).toBe(true)
+        })
+    })
+    test("Bad Queries do not effect outcome length or status", () =>{
+        return request(app)
+        .get('/api/articles?article_id=ga')
+        .then((res)=>{
+                expect(res.statusCode).toBe(200)
+                expect(res.body.articles.length > 0)
+        })
+    })
+  })
+describe("GET /api/articles/", () => {
```

Comment:
make sure this describe is accurately describing what endpoint you are going to be testing